.env

```diff
@@ -1,3 +1,11 @@
 JWT_SECRET='SECRET'
 NEXT_PUBLIC_RAPID_API_KEY='ac41cab2aamsh587d2717936bec9p185b22jsna20c36b1f5ef'
-NEXT_PUBLIC_FIREBASE_API_KEY='AIzaSyCiaPye-FbO69Lbh4algyKcOHGfG8IWsSg'
\ No newline at end of file
+NEXT_PUBLIC_FIREBASE_API_KEY='AIzaSyCiaPye-FbO69Lbh4algyKcOHGfG8IWsSg'
+NEXT_FIREBASE_AUTH_DOMAIN="truecrime-6f10f.firebaseapp.com"
+NEXT_PUBLIC_FIREBASE_DATABASE_URL="https://truecrime-6f10f-default-rtdb.europe-west1.firebasedatabase.app"
+NEXT_PUBLIC_FIREBASE_PROJECT_ID="truecrime-6f10f"
+NEXT_PUBLIC_FIREBASE_STORAGE_BUCKET="truecrime-6f10f.appspot.com"
+NEXT_PUBLIC_FIREBASE_MESSAGING_SENDER_ID=597261748863
+NEXT_PUBLIC_FIREBASE_APP_ID="1:597261748863:web:fd6b774208c75be6224d4a"
+NEXT_PUBLIC_FIREBASE_MEASUREMENT_ID="G-F2TE95EBVY"
+NEXT_PUBLIC_FIREBASE_SERVICE_ACCOUNT_KEY=firebase/serviceAccountKey.json
```

Comment:
@jgrgurovic this is still not fixed

Reply:
The .env file was pushed to GitHub in earlier commits. I've now removed it from the repository using git rm --cached .env    
 but the matter of the fact is that it can still be found in the earlier commits.  Then I added it to .gitignore, and configured environment variables in the Netlify dashboard.